astrokat/observe_main.py

```diff
@@ -313,6 +319,7 @@ def run_observation(opts, kat):
     obs_plan_params = opts.obs_plan_params
     # remove observation specific instructions housed in YAML file
     del opts.obs_plan_params
+    local_horizon = obs_plan_params['instrument']['horizon']
```

Comment:
Is the value also going to end up in CAM's session object?  E.g. with the `--horizon` option.  If not, the value in the yaml file could differ from session's meaning things like `session.track` fail or succeed unexpectedly.

Reply:
yaml does not support --horizon anymore -- will get value and assign to session to ensure basic functionality